build/Targets/Settings.props

```diff
@@ -22,6 +22,7 @@
     <RoslynPortableTargetFrameworks>net46;netcoreapp2.0</RoslynPortableTargetFrameworks>
  
     <Features>strict,IOperation</Features>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
```

Comment:
:memo: It is my understanding that this property alone doesn't really do anything. In order to truly test the end-to-end, we need to add the following as well:

```xml
<CompileUsingReferenceAssemblies>true</CompileUsingReferenceAssemblies>
```